pkgs/development/libraries/dirac/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "dirac-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/dirac/${name}.tar.gz";
+    sha256 = "1z803yzp17cj69wn11iyb13swqdd9xdzr58dsk6ghpr3ipqicsw1";
+  };
+
+  buildInputs = [ doxygen ];
+  enableParallelBuilding = true;
+
+  patches = [ ./dirac-1.0.2.patch ];
```

Comment:
Compiler flags are hardcoded in configuration script. Actually, I couldn't even compile the source without that patch due to critical errors similar to this:
```bash
quant_chooser.cpp:343:49: error: invalid operands of types ‘__gnu_cxx::__enable_if<true, double>::__type {aka double}’ and ‘int’ to binary ‘operator<<’
             if ( (std::abs(m_coeff_data[j][i])<<2) >= u_threshold )
```
I've tried the previous version with no luck.
It will be nice if dirac-research team accepts this patch, but it seems they dropped the project: https://sourceforge.net/p/dirac/discussion/353618/thread/45510d46/